test/packetCycleTests.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-env mocha */
+// Tests packet serialization/deserialization from with raw binary from minecraft-packets
+const { createSerializer, createDeserializer, states, supportedVersions } = require('minecraft-protocol')
+const mcPackets = require('minecraft-packets')
+const assert = require('assert')
+
+const serializers = {}
+const deserializers = {}
+const makeClientSerializer = version => createSerializer({ state: states.PLAY, version, isServer: true })
+const makeClientDeserializer = version => createDeserializer({ state: states.PLAY, version })
+
+describe('Packet cycle tests', () => {
+  for (const supportedVersion of supportedVersions) {
+    const mcData = require('minecraft-data')(supportedVersion)
+    const version = mcData.version.minecraftVersion
+    it('Packet cycle tests for ${version}', () => {
```

Comment:
Is that the right thing

Reply:
fixed, but now the grep isn't being called: https://github.com/PrismarineJS/node-minecraft-protocol/runs/4488855401?check_suite_focus=true#step:6:13